workshop/single-cell-resources.md

```diff
@@ -8,7 +8,9 @@ Please note, this is not an exhaustive list.
 It includes multiple types of resources for various topics in single-cell RNA-seq analysis, but does not represent the complete breadth of analysis topics.
 Resources are listed by topic and in alphabetical order, not in order of recommendation.
 
-**Table of contents**
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
```suggestion
**Table of Contents** 
```

Reply:
Just peeked at this, and we can do - https://github.com/thlorenz/doctoc#specifying-a-custom-toc-title

tested locally, and it needs the `**` - 
```
doctoc blah.md --title **Table of Contents**
```

Reply:
Thanks Stephanie! Should be fixed now using that trick.

Reply:
Once it is removed, it stays removed, so either way would have been fine!